website/docs/r/container_cluster.html.markdown

```diff
@@ -80,9 +80,9 @@ output "cluster_ca_certificate" {
     all specified zones.
 
 * `addons_config` - (Optional) The configuration for addons supported by Google
```

Comment:
Suggestion: At line 6 (sorry, can't comment on that line), use Google Kubernetes Engine (GKE). And below in the docs you can use GKE.

Reply:
Done